jormungandr/src/explorer/graphql/certificates.rs

```diff
@@ -1,14 +1,14 @@
 use super::error::ErrorKind;
+use async_graphql::*;
 use chain_impl_mockchain::certificate;
-use juniper::GraphQLUnion;
 use std::convert::TryFrom;
 
 use super::scalars::{PayloadType, PoolId, PublicKey, TimeOffsetSeconds, VotePlanId};
-use super::{Address, BlockDate, Context, ExplorerAddress, Pool, Proposal, TaxType};
-use juniper::FieldResult;
+use super::{Address, BlockDate, ExplorerAddress, Pool, Proposal, TaxType};
+use crate::rest::explorer::EContext as RestContext;
 
 // interface for grouping certificates as a graphl union
-#[derive(GraphQLUnion)]
+#[derive(Union)]
```

Comment:
```suggestion
#[derive(async_graphql::Union)]
```
This would look much more descriptive. However, this is a graphql service module, so it's OK just to
import the names explicitly in the top-level `use` statement so that the origin of these derive macros can be seen in the source file.